### DISCUSSION TITLE
Can several homes be read out via one account?

### DISCUSSION THREAD

**Discussion by Gilly79** [Ludy87/ecotrend-ista#131] [General]
We have a rental apartment over two levels and for this Ista has created two apartments under one account. Your integration reads out the data that I last looked at in the Ista Online Portal. Is there a way to take this into account in your integration so that both apartments can be read without me always having to switch back and forth in the online portal?

**Comment by Ludy87** [comment]
I would have to have a file to see if I can do something about it.

Google Chrome:
1. Go to https://ecotrend.ista.de/
2. Press F12 to open Chrome's console
3. go to the Network tab
4. Press Fetch/XHR
5. log in
6. look for the entry consumptions?consumptionUnitUuid=...... and account
7. right click
8. copy -> copy answer
9. save the contents into a file with the format consumptions_YOUR_GITHUB_NAME.json and account_YOUR_GITHUB_NAME.json
10. change your personal data - DO NOT DELETE - if there is data with the same content, then change it with the same changed data
11. send me the data

**Reply by Gilly79** [reply]
Hi Lugy87,
sorry for the late answer. I created the two files, but I don't know how to send them to you.
